cmd/prcompFormat/testdata/readme.txt

```diff
@@ -0,0 +1,3 @@
+Please see the vglDocumentation repository for a detailed explanation of
```

Comment:
Should this be in the usage message of the program?  My concern is that people trying to figure out how to run this, will not look for documentation in "testdata".  How extensive are the R commands, can you add them as a line or two in the usage message for this program?